test/extended/util/framework.go

```diff
@@ -2055,3 +2056,11 @@ func GetHypershiftManagementClusterConfigAndNamespace() (string, string, error)
 
 	return hypershiftManagementClusterKubeconfig, hypershiftManagementClusterNamespace, nil
 }
+
+func SkipIfExternalControlplaneTopolocy(oc *CLI, reason string) {
```

Comment:
```suggestion
func SkipIfExternalControlplaneTopology(oc *CLI, reason string) {
```

Reply:
Fixed. I will look for a vim spelling plugin that can deal with camelcase :upside_down_face: